lib/streamlit/report_session.py

```diff
@@ -413,19 +368,42 @@ def _enqueue_new_report_message(self):
 
     def _enqueue_report_finished_message(self, status):
         """Enqueue a report_finished ForwardMsg.
-
         Parameters
         ----------
         status : ReportFinishedStatus
-
         """
         msg = ForwardMsg()
         msg.report_finished = status
         self.enqueue(msg)
 
+    def handle_git_information(self):
```

Comment:
Nit: To be consistent with e.g. `handle_rerun_script_request`, would like to call this `handle_git_info_request`